style.css

```diff
@@ -518,41 +520,99 @@ input:hover {
     display: none;
   }
 
-  .bar {
-    width: 30px;
-    height: 4px;
-    border-radius: 3px;
-    background: black;
-    margin: 5px;
+  .main-menu i {
+    font-size: 20px;
+    position: absolute;
+    right: 2%;
   }
 
   .body p {
     width: 95%;
     margin-left: 10px;
   }
 
-  .h-menu a {
+  .greet {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 20px;
+    line-height: 28px;
+    color: #28352f;
+    position: absolute;
+    left: 2%;
+  }
+
+  .hidden-menu {
+    display: none;
+    width: 100vw;
+    height: 100vh;
+    background: #e5e5e5;
+    overflow: hidden;
+  }
+
+  .hid-menu {
+    border: 1px solid white;
+    height: 90vh;
+    background-color: white;
+    border-radius: 0 0 20px 20px;
+  }
+
+  .hidden-menu hr {
+    margin-left: 35%;
+    width: 134px;
+    height: 4px;
+    background: #0d151e;
+    border-radius: 10px;
+    position: absolute;
+    bottom: 0;
+  }
+
+  .hidden-cont {
+    position: absolute;
+    left: 2%;
+    top: 10%;
     font-family: 'Inter', sans-serif;
     font-style: normal;
     font-weight: 600;
     font-size: 20px;
     line-height: 28px;
+    list-style: none;
+  }
+
+  .hidden-cont li {
+    margin-top: 30px;
+  }
+
+  .hidden-cont a {
     color: #28352f;
+    font-size: 32px;
   }
 
   .pointer {
     margin-top: 80px;
   }
 
-  .h-menu a:hover {
+  .greet:hover {
     color: blue;
     cursor: pointer;
     transform: scale(1.1);
   }
 
-  .bar:hover {
+  .hidden-cont a:hover {
+    cursor: pointer;
+    color: blue;
+  }
+
+  .close-btn {
+    position: absolute;
+    right: 5%;
+    font-size: 2rem;
     cursor: pointer;
-    background: blue;
```

Comment:
- [ ] Kindly adjust this toa  background: `#67798E` as given on the figma template